src/UserManager.ts

```diff
@@ -163,8 +165,7 @@ export class UserManager {
         const user = await this._signinEnd(url);
         if (user.profile && user.profile.sub) {
             logger.info("success, signed in subject", user.profile.sub);
-        }
-        else {
```

Comment:
please avoid changing code style, i wont merge this kind of changes

Reply:
I am sorry, this is something that my IDE does if not otherwise configured. Perhaps you should consider enforcing this specific rule as well via the .editorconfig file if possible.